### PR TITLE
US_FL_jacksonville

### PR DESCRIPTION
**Pull request by ManuBenito259**

### CODE REVIEW

File: regions/us-fl-jacksonville.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      },
+      {
+        "lat": 30.101103357350834,
+        "lng": -81.61897659301758,
+        "name": "Fruit Cove"
+      }
+    ],
+    "polygon": {
+      "type": "Polygon",
+      "coordinates": [
```
Comment (fersagui) [inline regions/us-fl-jacksonville.json:32]: @ManuBenito259 could you change this polygon to be next to the actual US_FL_Orlando polygon? 
Screen Shot 2022-10-05 at 16 08 37

File: regions/us-fl-jacksonville.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            29.80013454835905
+          ]
+        ]
+      ]
+    }
+  },
+  "currency": "USD",
+  "trafficRule": "right",
+  "feeds": [
+    "us-jta"
```
Comment (fersagui) [inline regions/us-fl-jacksonville.json:61]: Could you check if there are not other sources of data on this area? I have found this one too: https://transitfeeds.com/l/407-st-augustine-fl-usa

**Review by fersagui — CHANGES_REQUESTED** [review]

File: regions/us-fl-jacksonville.json (hunk 1 above)
Comment (ManuBenito259) [reply regions/us-fl-jacksonville.json:32]: On it, should they overlap?

Comment (fersagui) [reply regions/us-fl-jacksonville.json:32]: yes, just a few. We could add services that connects both areas if there are any.

File: regions/us-fl-jacksonville.json (hunk 2 above)
Comment (ManuBenito259) [reply regions/us-fl-jacksonville.json]: imagen


Only JTA and St. Augustine's Sunshine Bus according to transit land

**Review by fersagui — APPROVED** [review]
LGTM

File: regions/us-fl-jacksonville.json (hunk 2 above)
Comment (fersagui) [reply regions/us-fl-jacksonville.json]: Ok, thanks for the feedback

**Pull request merged**